tools/sdk-generation-pipeline/pipelines/steps/generate-sdk.yml

```diff
@@ -0,0 +1,102 @@
+parameters:
```

Comment:
@zzvswxy where do we end up hosting the pipeline for this yml? Do we hook-up the pipeline from this repo or somewhere else? I suspect we will want this to eventually be moved or at least consumed from the azure-rest-api-specs repo itself correct?

cc @raych1 @chunyu3

Reply:
Hi @weshaggard , 
Question: where do we end up hosting the pipeline for this yml? 
Answer: https://github.com/Azure/azure-sdk-tools/blob/main/tools/sdk-generation-pipeline/sdk-generation-pipeline.yml is the basic pipeline yml file. And it called generated-sdk.yml
Q: Do we hook-up the pipeline from this repo or somewhere else?
A: Azure/azure-sdk-tools hook-up the pipeline.
Q:  I suspect we will want this to eventually be moved or at least consumed from the azure-rest-api-specs repo itself correct?
A: For now, the pipeline is consumed by E2E power apps 
![image](https://user-images.githubusercontent.com/20081898/184842030-18f6c084-85e5-4263-aeba-25fb7ca5d5bc.png)

cc @dw511214992

Reply:
Can you please link to the pipeline in internal? 

I agree it is used for the E2E power apps but we will want to consume it from the specs PR process at some point.

Reply:
Leave you a message on teams as it's a internal project pipeline.
Thanks